### PR TITLE
ADD: description of the build workflow

### PR DESCRIPTION
**Pull request by kmuehlbauer**
As suggested in #65 this is adds some information on the build workflow.

### CODE REVIEW

**Comment by mgrover1** [comment]
Thanks @kmuehlbauer for putting this together!! The docker building process is super clear! I think the main things still missing are:
- [ ] Mention of JupyterBook and the Pythia Cookbook template
- [ ] How to add an individual new package to the workflow
- [ ] Health checks and how the notebooks are all executed

File: introductions/getting-started.md [hunk 1]
```diff
@@ -62,6 +51,9 @@ $ repo2docker .
 
 This will fetch the docker image from ghcr and fire up a container running jupyterlab.
 
+### Health check
```
Comment (mgrover1) [inline introductions/getting-started.md:54]: Perhaps link to the GitHub action here (or at least the badge)

File: introductions/getting-started.md [hunk 2]
```diff
@@ -70,3 +62,34 @@ This will fetch the docker image from ghcr and fire up a container running jupyt
...
+## Using the PANGEO Cloud with Binder
+
+### 1. Make Sure You Have a Github Account
+The first step is to make sure you have a Github Account.
+
+Here is the link if you do have one already:
+- [Github account creation link](https://github.com/join)
+
+### 2. Log into Pangeo Binder
+Next, sign in and authenticate the Pangeo Binder, which is the platform we will use for the workshop:
```
Comment (mgrover1) [inline introductions/getting-started.md:76]: I think we should mention the machine limitations here (10 GB of memory, 2 core CPU, etc.) 

And mention if they are interested in using a bigger instance, they can follow the instructions here - https://pangeo.io/cloud.html

Or at a minimum, link back to the Pangeo docs here.

File: introductions/getting-started.md [hunk 3]
```diff
@@ -1,38 +1,27 @@
...
-If you are having issues with that (ex. it is taking a long time), try using the following link:
-```
-https://hub.aws-uswest2-binder.pangeo.io/user/{your_github_username}/lab
-```
-Where you replace `{your_github_username}` with your github username (ex. `mgrover1`)
+If there is a package missing for your use-case you can just add the package to [binder/environment.yml](https://github.com/openradar/erad2022/blob/main/binder/environment.yml) and activate a new build (see below).
+Notebooks might be added to the [notebooks-folder](https://github.com/openradar/erad2022/tree/main/notebooks).
 
-## Description of the build workflow
+## Build workflow
```
Comment (mgrover1) [inline introductions/getting-started.md:14]: Perhaps adding a "general philosophy" here would be helpful. For example,

"We work on pre-building the environment and toolkit so you can deploy this on the platform of your choice! Making the Open Radar Science stack available across different environments is essential to ensure reproducibility. We take advantage of ["containerization"](https://www.ibm.com/cloud/learn/containerization), which enables packaging all of these tools together so you can deploy it on the platform of your choice. In this case, we are deploying a software stack which includes a variety of different languages (ex. Python, C++, C) and deploy it on a cloud platform. We use [Docker](https://www.docker.com) as our main containerization tool."

Feel free to suggest other overarching ideas,

**Review by mgrover1 — COMMENTED** [review]
A couple more changes suggested... great stuff so far!!

**Comment by kmuehlbauer** [comment]
Great suggestions @mgrover1. Feel free to push to my branch.

**Comment by mgrover1** [comment]
@kmuehlbauer I went ahead and added those suggestions. Let me know what you think!

**Comment by kmuehlbauer** [comment]
@mgrover1 Yes, that's very nice read. I'll fix some typos and links later.

**Review by mgrover1 — APPROVED** [review]
Looks great! Good to merge @kmuehlbauer ?

**Pull request merged**